core/jvm/src/test/scala/scalaz/zio/RTSSpec.scala

```diff
@@ -958,6 +959,15 @@ class RTSSpec(implicit ee: ExecutionEnv) extends AbstractRTSSpec {
 
   }
 
+  def testBlockingIOIsInterruptible = unsafeRun(
+    for {
+      done  <- Ref(false)
+      fiber <- IO.blocking(while(true){scala.io.StdIn.readLine()}).ensuring(done.set(true)).fork
+      _     <- fiber.interrupt
```

Comment:
Is it possible that test failed because docs of ensuring states:
```
This method should not be used for cleaning up resources, because it's possible the fiber will be interrupted after acquisition but before the finalizer is added.
```
?